lib/tasks/audit.rake

```diff
@@ -7,5 +7,31 @@ task :audit, %i[collection_druid] => :environment do |_task, args|
   puts Audit::WarcAuditer.audit(collection_druid: collection.druid,
                                 wasapi_collection_id: collection.wasapi_collection_id,
                                 wasapi_account: collection.wasapi_account,
-                                wasapi_provider: collection.wasapi_provider)
+                                wasapi_provider: collection.wasapi_provider,
+                                embargo_months: collection.embargo_months)
+end
+
+desc 'Remediate missing WARCs based on audit results'
+task :remediate, %i[collection_druid] => :environment do |_task, args|
+  collection = Collection.find_by(druid: args[:collection_druid])
+  puts "Auditing #{collection.title}. This might take a minute."
+  filenames = Audit::WarcAuditer.audit(collection_druid: collection.druid,
+                                       wasapi_collection_id: collection.wasapi_collection_id,
+                                       wasapi_account: collection.wasapi_account,
+                                       wasapi_provider: collection.wasapi_provider,
+                                       embargo_months: collection.embargo_months)
```

Comment:
It is possible/likely that this will be used in the future in cases in which there is not a collection active record object, so I'd be inclined to leave it as it is (though it does makes the number of params awkward).